css/doc.scss

```diff
@@ -32,3 +32,25 @@ permalink: /css/doc.css
 		padding: 2px 7px 0;
 	}
 }
+
+@media screen and (max-width: 480px) {
```

Comment:
I'd like the breakpoint to be a variable, so that it can be used consistently in any future changes.

Reply:
Noted, will do.